swift-chat-app/LoginView.swift

```diff
@@ -72,16 +89,42 @@ struct LoginView: View {
     
     private func handleActionButton() {
         if isLoginMode {
-            print("Should login into Firebase")
+            loginUser()
         } else {
-            print("Should create new account")
+            createNewAccount()
+        }
+    }
+    
+    private func loginUser() {
```

Comment:
I wonder if we should move the login and account creation functionality to an AccountService?

Reply:
That makes sense to me

Reply:
Okay maybe we can merge in this PR and I will put a PR with that idea?